tests/testthat/_snaps/write-docker.md

```diff
@@ -0,0 +1,61 @@
+# create plumber.R with packages
+
+    Code
+      cat(readr::read_lines(file.path(tmp_dir, "Dockerfile")), sep = "\n")
+    Output
+      # Generated by the vetiver package; edit with care
+      
+      FROM rstudio/plumber:latest
+      ARG R_VERSION=<r_version>
+      ENV R_VERSION=$R_VERSION
```

Comment:
Pretty sure this doesn't work 😕

Reply:
Correct. :-(

Two options I can think of:
1. Build the `rstudio/plumber` Docker image and save it as a new custom name. Then use the custom name in the vetiver image. This would reduce installations of plumber, but I'm not too worried about that.
2. Make a combo Docker image as `rstudio/plumber` is not that big can could be handled by `pak`. 

I like second solution. I'll comment on the earlier conversation about implementation